contracts/contracts/SimpleWriteAccessController/library.cairo

```diff
@@ -0,0 +1,142 @@
+%lang starknet
```

Comment:
@archseer are there other contracts as candidates to be restructured like this, to enable composability later?

Reply:
I don't think so, I already extracted `Ownable`